pages/linux/pactl.md

```diff
@@ -0,0 +1,23 @@
+# pactl
+
+> Control a running PulseAudio sound server.
+
+- List all sinks (or other types - sinks are outputs and sink-inputs are active audio streams):
+
+`pactl list {{sinks}} short`
+
+- Change the default sink (output) to 1 (the number can be retrieved via the list command):
```

Comment:
Please wrap "list" in backticks since it's a specific keyword for this command.

Reply:
Inside the description you mean? It makes sense, but I can't find anything about this is the style guidelines.

Come to think of it, would it perhaps be better to change "via the `list` command" to "via the `list` subcommand"?

Reply:
Yeah, there have been a couple discussions about usage of backticks and requests to have a guideline added to the documentation. I'll add it to my list, but I'm afraid it may take me a while to get to it :/

In any case, yes, that's what I meant. And I agree with your suggestion to use "subcommand" instead. 👍